db/post_migrate/20210530122334_move_existing_triggers_to_fields.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class MoveExistingTriggersToFields < ActiveRecord::Migration[6.1]
+  def create_field(automation, component, name, metadata)
+    automation.fields.create!(
```

Comment:
Same here, how hard is it to not use AR?

Reply:
done in https://github.com/discourse/discourse-automation/pull/21/commits/d05453fcda70b9470cbe4b8daa6f11b223e24fa5